Documentation.md

```diff
@@ -105,9 +110,23 @@ and made available to your JS client code.
 * `ccpUrl`: The URL of the CCP.  This is the page you would normally navigate to
   in order to use the CCP in a standalone page, it is different for each
   instance.
-* `region`: Amazon connect instance region. ex: us-west-2 for PDX ccp instance.  only required for chat channel.
-* `loginPopup`: Optional, defaults to `true`.  Set to `false` to disable the login
-  popup which is shown when the user's authentication expires.
+* `region`: Amazon connect instance region. ex: us-west-2 for PDX ccp instance. 
+   Only required for chat channel.
+* `loginPopup`: Optional.  Set to `false` to disable the login popup which is 
```

Comment:
Can you keep the `defaults to true` language? We don't want this to be a breaking change.

Reply:
Aren't you worried that that language will be confusing? While this isn't a breaking change and setting the value to "true" will still accept all default values, it is not accurate to say that the default is not "true." I called out what the defaults are for each option.

Reply:
As mentioned by Haochi, could you keep the `Optional, defaults to true.`